doc/api/http2.md

```diff
@@ -524,6 +524,33 @@ Used to set a callback function that is called when there is no activity on
 the `Http2Session` after `msecs` milliseconds. The given `callback` is
 registered as a listener on the `'timeout'` event.
 
+#### http2session.setConnectionWindowSize(windowSize)
```

Comment:
Would a getter and setter be better for an API?

Reply:
It's a good point, I am not sure to be able to detect it. There is not acknoledge on a SETTINGS_INITIAL_WINDOW_SIZE
But, we can add a test for testing the limit of the buffer allocation for instance and insure that code results are covered.

Reply:
I will add the getter